.storybook/main.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  stories: [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
I guess these two lines could be shortened to `../src/**/*.stories.@(js|jsx|mdx|ts|tsx)`?

Reply:
those where generated, but indeed you are right